build/edgesite/edgesite-agent.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: v1
+kind: Pod
```

Comment:
Do we need to use deployment?

Reply:
now we only use one pod, no need to use deployment

Reply:
Please use deployment, thanks.